tidb-cloud/create-tidb-cluster.md

```diff
@@ -92,9 +93,9 @@ To create a Serverless Tier cluster, take the following steps:
 
 To create a Dedicated Tier cluster, take the following steps:
 
-1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page.
 
-2. Click **Create Cluster**.
+2. If you have multiple projects, choose a target project in the left navigation pane, and then click **Create Cluster** in the upper-right corner. Otherwise, click **Create Cluster** in the upper-right corner directly.
```

Comment:
```suggestion
2. If you have multiple projects, switch to the target project in the left navigation pane, and then click **Create Cluster** in the upper-right corner. Otherwise, click **Create Cluster** in the upper-right corner directly.
```